examples/soap-calculator/src/datasources/calculator-ds.datasource.ts

```diff
@@ -0,0 +1,21 @@
+import {inject} from '@loopback/core';
+import {juggler, AnyObject} from '@loopback/repository';
+const config = require('./calculator-ds.datasource.json');
+
+export class CalculatorDSDataSource extends juggler.DataSource {
```

Comment:
Similarly here, can we omit `DS` from the class name and use `CalculatorDataSource`?